predict.py

```diff
@@ -294,5 +334,30 @@ def main(args):
         action="store_true",
         help="Whether the structure is NMR. NMR will have different states so TIMED will try to build a consensus",
     )
+
+    parser.add_argument(
+        "--res_to_fix",
+        type=parse_residue_list,
+        help="Specify the residue numbers that need to be fixed, separated by commas. Example: '2 3 4 5, 6 7 8' will keep 2 3 4 5 6 7 8th residues as identical with WT and predict the rest.",
+    )
+
+    parser.add_argument(
+        "--chains_to_fix",
+        type=parse_chain_list,
+        help="Specify the chains need to be fixed, separated by commas. Example: 'A, B' ",
+    )
+
+    parser.add_argument(
+        "--chains_to_predict",
+        type=parse_chain_list,
+        help="Specify the chains need to be predicted, separated by commas. Example: 'A, B' ",
+    )
+
```

Comment:
Imo this is too many arguments which inherently make the code harder to understand. 

Let's stick with the "to_fix' and for the moment predict anything else.

Also I think it should be only one input argument rather than two. 

like


```
--res_to_fix A: 5, 10, 22; B: 2, 23, 1
```